tati/bio.md

```diff
@@ -9,3 +9,4 @@ Para de ser *BOCO*
 ROSA e AZUL
 tutu
 
+uahhasuhsauhsavfdfgd
```

Comment:
What do you think about delete this line

Reply:
Good ideia

Reply:
[https://giphy.com/gifs/26tP1TiiRIF0rLhWU]

Reply:
![I Dont Think So No Way GIF](https://giphy.com/gifs/26tP1TiiRIF0rLhWU)